### DISCUSSION TITLE
How to write a JSON to map Salt Edge fields?

### DISCUSSION THREAD

**Discussion by tiagodj** [firefly-iii/firefly-iii#5756] [Firefly III] [answered]
Hi,
I've finished setting up the data importer, and now I am able to successfully import transactions from my bank.
The issue I have at the moment is how to configure the mapping to work properly.
The JSON documentation talks about using column numbers, but I think that is meant to the CSV file import. I see nothing similar to configure the "mapping" section for Spectre fields.
Am I missing something?

Thanks in advance for this amazing tool!

**Comment by JC5** [comment, marked as answer]
You can use the UI to map values; the resulting JSON will contain the mappings. From the top of my head there are only two fields you can map: the accounts are in column 0 and the categories are in column 1.
